spec/lib/rmt/cli/export_spec.rb

```diff
@@ -89,9 +90,9 @@
             FileUtils.mkdir_p path
             File.write("#{path}/repos.json", repo_ids.to_json)
 
+            expect(RMT::Mirror).to receive(:from_repo_model).exactly(repo_ids.count).times.and_return(mirror_double)
             expect(mirror_double).to receive(:mirror)
             expect(mirror_double).to receive(:mirror).and_raise(RMT::Mirror::Exception, 'black mirror')
```

Comment:
Still a duplicate `expect(mirror_double).to receive(:mirror)` (lines 94 & 95), is that intentional?